spec/unit/helpers/check_helper_production_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'gooddata/lcm/actions/base_action'
+require 'gooddata/lcm/helpers/check_helper'
+require 'gooddata/lcm/types/types'
+
+describe 'GoodData::LCM2::Helpers::Check' do
+  let(:params) do
+    params = {
+      test_param_two: 'Testing param two',
+      test_param_three: 'Testing param three',
+      test_param_four: 4,
+      UPPER_case_param: 'qux'
+    }
+    GoodData::LCM2.convert_to_smart_hash(params)
+  end
+  context 'when running outside the tests' do
+    before {
+      def ENV.[](a)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `a`. If it's necessary, use `_` or `_a` as an argument name to indicate that it won't be used. You can also write as `[](*)` if you want the method to accept any arguments but don't care about them.